cmd/rook/ceph/admission.go

```diff
@@ -36,11 +50,12 @@ func init() {
 
 func startAdmissionController(cmd *cobra.Command, args []string) {
 	rook.SetLogLevel()
-
 	rook.LogStartupInfo(admissionCmd.Flags())
 
-	context := rook.NewContext()
-
-	a := admission.New(context, "ceph", operator.ValidateCephResource)
+	err := cephv1.AddToScheme(scheme)
+	if err != nil {
+		logger.Errorf("failed to add to scheme %v. ", err)
```

Comment:
If there is a failure we need to exit since the admission controller would fail to start later anyway. See other places where we call 
```
rook.TerminateFatal(err)
```